lib/DAV/WorkspacePlugin.php

```diff
@@ -93,24 +93,23 @@ public function propFind(PropFind $propFind, INode $node) {
 		if (!$workspaceAvailable || !$workspaceEnabled) {
 			return;
 		}
-
-		$propFind->handle(self::WORKSPACE_PROPERTY, function () use ($node) {
-			/** @var Folder[] $nodes */
-			$nodes = $this->rootFolder->getUserFolder($this->userId)->getById($node->getId());
-			if (count($nodes) > 0) {
-				/** @var File $file */
-				try {
-					$file = $this->workspaceService->getFile($nodes[0]);
-					if ($file instanceof File) {
-						return $file->getContent();
-					}
-				} catch (StorageNotAvailableException $e) {
+		if ($propFind->getDepth() > 0){
+			$propFind->handle(self::WORKSPACE_PROPERTY, function () use ($node) {
+				/** @var Folder[] $nodes */
+				$nodes = $this->rootFolder->getUserFolder($this->userId)->getById($node->getId());
+				if (count($nodes) > 0) {
+          /** @var File $file */
```

Comment:
Please use tabs :wink:

Reply:
I was wondering why it looks so odd, now it should be ok

Reply:
Thanks seems good now :+1: